JS/Lesson15/emrah/script.js

```diff
@@ -1,164 +1,201 @@
 // 1. Use the `length` property to find the length of a string and print it to the console.
-const stringExercise1 = 'Object-oriented programming is a powerful paradigm.';
+const stringExercise1 = "Object-oriented programming is a powerful paradigm.";
 
-console.log('Exercise1: ', stringExercise1.length);
+console.log("Exercise1: ", stringExercise1.length);
 
-console.log('------------------------');
+console.log("------------------------");
 // 2. Use the `slice` method to extract 'Debugging code' from the string below and print
 // it to the console.
 const stringExercise2 =
-	'Debugging code can be a frustrating but necessary task.';
+  "Debugging code can be a frustrating but necessary task.";
 
-console.log('exercise2: ', stringExercise2.slice(0, 14));
+console.log("Exercise2: ", stringExercise2.slice(0, 14));
 
-console.log('------------------------');
+console.log("------------------------");
 // 3. Use the `toUpperCase` method to convert a string to uppercase and print it to the console.
 const stringExercise3 =
-	'Python is a popular language for data analysis and machine learning.';
-console.log('exercise3: ' /* Result here */);
-console.log('------------------------');
+  "Python is a popular language for data analysis and machine learning.";
+console.log("Exercise3: ", stringExercise3.toUpperCase());
+console.log("------------------------");
 // 4. Use the `toLowerCase` method to convert a string to lowercase and print it to the console.
 const stringExercise4 =
-	'FUNCTIONAL PROGRAMMING EMPHASIZES PURE FUNCTIONS AND IMMUTABLE DATA.';
-console.log('Exercise4: ' /* Result here */);
+  "FUNCTIONAL PROGRAMMING EMPHASIZES PURE FUNCTIONS AND IMMUTABLE DATA.";
+console.log("Exercise4: ", stringExercise4.toLowerCase());
 
-console.log('------------------------');
+console.log("------------------------");
 // 5.a Use the `indexOf` method to find the index of "l" character in the string stringExercise5
 // and print it to the console.
 const stringExercise5 =
-	'JavaScript is a versatile language used for web development and beyond.';
+  "JavaScript is a versatile language used for web development and beyond.";
 
-console.log('Exercise5: ', stringExercise5.indexOf('l'));
+console.log("Exercise5.a: ", stringExercise5.indexOf("l"));
 
-console.log('------------------------');
+console.log("------------------------");
 // 5.b Use the `indexOf` method to find the index of "language" in the string stringExercise5 and print it to the console.
 
-console.log('Exercise5.b: ', stringExercise5.indexOf("language"));
-console.log('------------------------');
+console.log("Exercise5.b: ", stringExercise5.indexOf("language"));
+console.log("------------------------");
 // 6. Use the `lastIndexOf` method to find the last index of of "l" character in the string stringExercise5 and print it to the console.
 
-console.log('Exercise6: ' /* Result here */);
-console.log('------------------------');
+console.log("Exercise6: ", string1Exercise5.lastIndexOf(l));
```

Comment:
Should be like this instead:
```suggestion
console.log("Exercise6: ", stringExercise5.lastIndexOf("l"));
```
as variable `string1Exercise5` is not declared anywhere here.
Also, don't forget quotes for the string that you're searching, otherwise compiler is looking for a variable named `l`